### ISSUE TITLE
Travel time by transit from each zone to another

### ISSUE THREAD

**Issue by FarnooshNm** [matsim-org/matsim-code-examples#768]
I have a network with a number of zones and I want to find the travel time by transit (separated by access time, egress time, in-vehicle time, number of transfers, and transfer time) from each zone to other zones. It seems I should use TransitRouter but I'm not sure how it should be done. I'd be grateful for any information or resources in this regard. 
Thank you.

**Comment by FarnooshNm** [comment]
I ended up using the [Skim Matrices](https://github.com/SchweizerischeBundesbahnen/matsim-sbb-extensions#skim-matrices) extension for MATSim provided by Swiss Federal Railways.

**Comment by jingjunL** [comment]
Hi @FarnooshNm , I would like to use the skim matrices extension as well but cannot configure it properly... I am wondering whether there is any code that you use online so that I can see it as a reference? Thanks :)

**Comment by FarnooshNm** [comment]
Hi @jingjunL,
Just to note, I didn't integrate it with matsim simulation. Instead, I used it to extract travel times by transit for different OD pairs. I think I used this code and made some modifications for my use case:
[CalculateSkimMatrices](https://github.com/SchweizerischeBundesbahnen/matsim-sbb-extensions/blob/master/src/main/java/ch/sbb/matsim/analysis/skims/CalculateSkimMatrices.java)
There is a main method at the bottom.
Hope that helps!
